app/views/api_docs/reference/reference.html.erb

```diff
@@ -113,3 +117,22 @@
 <% @api_reference.schemas.each do |schema| %>
   <%= render 'schema', schema: schema %>
 <% end %>
+
+<h2 class="govuk-heading-l" id="field-lengths">Field lengths summary</h2>
+
+<table class="govuk-table">
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header">Rule</th>
```

Comment:
What does rule mean here? Is there a possibility for different rules to be included here, or will it only ever be maxLength ones? Is it worth changing the heading to "Field" and trimming off the `maxLength` from all the values

<img width="979" alt="Screenshot 2021-02-09 at 08 46 25" src="https://user-images.githubusercontent.com/30071178/107338313-4daff700-6ab3-11eb-8369-4145bc7f08ca.png">

Reply:
Yeah, the rule is what the limit is referring to - and it would account for a minLength if one does show up later on. We currently don't have any mininmums though, so I can go ahead and trim off the the tail ends

Reply:
In that case maybe it's safer to leave it as-is. I'd be worried we might add a minLength one and start broadcasting it as a max length because of this table. Could we put the max/min part of the information in the Limit column maybe? Like:

Property | Limit
name | 2 min, 100 max
id | 10 max

Reply:
I might leave it as is, but add a ticket that lets us know this is a bit of a landmine, and at the point where we do have minimum lengths showing up, this will be a bit odd